cmd/cluster-olm-operator/main.go

```diff
@@ -149,6 +150,19 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		cc.EventRecorder.ForComponent("OLMIncompatibleOperatorController"),
 	)
 
+	// Side Effect! Update our environment immediately
+	err = controller.UpdateProxyEnvironment(klog.FromContext(ctx).WithName("main"), cl.ProxyClient)
+	if err != nil {
+		return err
+	}
+
+	proxyController := controller.NewProxyController(
+		"OLMProxyController",
```

Comment:
would it be worth moving all these controller ids to constants?

Reply:
I'll do new ones, but I don't want to go and modify code outside the code of the proxy work.